pkg/bean/app.go

```diff
@@ -506,6 +506,8 @@ type CDPipelineConfigObject struct {
 	TeamId                        int                                    `json:"-"`
 	EnvironmentIdentifier         string                                 `json:"-" `
 	IsVirtualEnvironment          bool                                   `json:"isVirtualEnvironment"`
+	PreDeployStage                *bean.PipelineStageDto                 `json:"preDeployStage,omitempty"`
```

Comment:
who is it diff from ``PreStage`` or ``PostStage`` ?? if the diff is new vs old, then we need to mark deprecated flag

Reply:
This will accept pre or post cd stage in steps format, rather than in yaml type which was accepted by PreStage type